js/resources.js

```diff
@@ -1,40 +1,28 @@
 $(document).ready(function(){
 
-  // Expand all
-  $( '.floe-resources-show' ).click(function() {
-      $( '.floe-resources-toggle' )
-        .attr('aria-expanded', true)
+  var toggleAll = function(elm, state) {
```

Comment:
I think I'd rename this since it no longer only toggles all - it's also used for toggling individual sections.